web/src/components/tasks/business-formation/BusinessFormationPaginator.tsx

```diff
@@ -48,12 +48,33 @@ export const BusinessFormationPaginator = (): ReactElement => {
 
   type ConfigFormationFields = keyof typeof Config.formation.fields;
 
+  const determineStepsWithErrors = (overrides?: {
+    hasSubmitted: boolean;
+  }): { stepsWithErrors: StepperStep[]; stepStates: StepperStep[] } => {
```

Comment:
[dust] Is there a better name for this function? I would imagine that `determineStepsWithErrors` would give me `stepsWithErrors`, but we get a complex object back that includes `stepStates` as well.

Reply:
Changed to determineStepsStates